.travis.yml

```diff
@@ -47,12 +47,16 @@ x_base_steps:
       - docker
 
 stages:
-  - check
-  - test
+  - name: check
+    if: type == pull_request
+  - name: test
+    if: type == pull_request
```

Comment:
Will we no longer run the check/test on master?

Reply:
Correct. We already run PR tests, so testing master post-merge is redundant.

Reply:
I only think that we need to run the test/check when we push the release tag. I mean to generate the tag. Otherwise, +1 for me.  However, in the latest release meeting, this idea was not accepted. @joelanford @jmrodri are you ok with as well